src/expressions/OrderByExpression.ts

```diff
@@ -4,13 +4,13 @@ import ISequence from './dataTypes/ISequence';
 import isSubtypeOf from './dataTypes/isSubtypeOf';
 import sequenceFactory from './dataTypes/sequenceFactory';
 import { getPrimitiveTypeName } from './dataTypes/typeHelpers';
-import Value, { SequenceMultiplicity, ValueType } from './dataTypes/Value';
+import Value, { ValueType } from './dataTypes/Value';
 import DynamicContext from './DynamicContext';
 import ExecutionParameters from './ExecutionParameters';
 import Expression, { RESULT_ORDERINGS } from './Expression';
 import FlworExpression from './FlworExpression';
 import convertItemsToCommonType from './functions/convertItemsToCommonType';
-import valueCompare from './operators/compares/valueCompare';
+import valueCompareFunction from './operators/compares/valueCompare';
```

Comment:
I usually try to keep filenames and the default export the same. Can you try to keep them the same here as well?

Reply:
Renamed in the upcoming commits